lib/config/configuration.go

```diff
@@ -1060,6 +1062,41 @@ func applyDatabasesConfig(fc *FileConfig, cfg *service.Config) error {
 	return nil
 }
 
+// readCACert reads database CA certificate from the config file.
+// First 'tls.ca_cert_file` is being read, then deprecated 'ca_cert_file' if
+// the first one is not set.
+func readCACert(database *Database) ([]byte, error) {
+	var (
+		caBytes []byte
+		err     error
+	)
+	if database.TLS.CACertFile != "" {
+		caBytes, err = os.ReadFile(database.TLS.CACertFile)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+	}
+
+	// ca_cert_file is deprecated, but we still support it.
+	// Print a warning if the old field is still being used.
+	if database.CACertFile != "" {
+		if database.TLS.CACertFile != "" {
+			// New and old fields are set. Ignore the old field.
+			log.Warn("Ignoring deprecated ca_cert_file; using tls.ca_cert_file.")
+		} else {
+			// Only old field is set, inform about deprecation.
+			log.Warn("ca_cert_file is deprecated, please use tls.ca_cert_file instead.")
```

Comment:
Similar to above, let's include the database name to help identify problematic config section.

Reply:
Added